applications/DemStructuresCouplingApplication/custom_utilities/dem_structures_coupling_utilities.h

```diff
@@ -217,14 +221,17 @@ void ComputeSandProductionWithDepthFirstSearchNonRecursiveImplementation(ModelPa
     static const double initial_total_mass_in_grams = current_total_mass_in_grams;
     const double cumulative_sand_mass_in_grams = initial_total_mass_in_grams - current_total_mass_in_grams;
 
-    ModelPart::ConditionsContainerType::iterator condition_begin = outer_walls_model_part.ConditionsBegin();
-    const double face_pressure_in_psi = condition_begin->GetValue(POSITIVE_FACE_PRESSURE) * 0.000145;
+    //ModelPart::ConditionsContainerType::iterator condition_begin = outer_walls_model_part.ConditionsBegin();
+    //const double face_pressure_in_psi = condition_begin->GetValue(POSITIVE_FACE_PRESSURE) * 0.000145;
+    //ProcessInfo& r_process_info = dem_model_part.GetProcessInfo();
+    ProcessInfo& r_process_info = dem_model_part.GetProcessInfo();
+    const double face_pressure_in_psi = fabs(r_process_info[TARGET_STRESS_Z]) * 0.000145;
```

Comment:
can we put this 0.000145 in the processInfo somehow? Any hardcoded number is prone to error in the future.